### DISCUSSION TITLE
启动器报错 4214

### DISCUSSION THREAD

**Discussion by awaae001** [Melledy/Grasscutter#1758] [Support]
如题

还有，那个证书的密码是什么？

**Comment by paranosan** [comment]
密码可能是123456，但这个不重要

**Reply by awaae001** [reply]
谢谢，

**Comment by awaae001** [comment]
已解决，4214问题打补丁即可解决

https://moeyy.cn/down/%E5%8E%9F%E7%A5%9E%E6%96%87%E4%BB%B6/%E8%A1%A5%E4%B8%81
文件放在
Genshin Impact\Genshin Impact Game\GenshinImpact_Data\Managed\Metadata

记得备份原补丁

**Reply by xinruowuqing** [reply]
试了下没有用唉 ，还是会报错4214 搞不明白了

**Reply by awaae001** [reply]
国服会拦截地址

**Reply by awaae001** [reply]
国际不会
